### PR TITLE
Update Riseup and Bitmask rules

### PR DESCRIPTION
**Pull request by just1602**
Updates subdomains and now use the new rules style.

### CODE REVIEW

**Comment by J0WI** [comment]
They do also exist:
www.demo.bitmask.net
nicknym.demo.bitmask.net
wallaby.demo.bitmask.net
api.demo.bitmask.net

Could you document mismatches on:
cdev.bitmask.net
unstable.bitmask.net

Sadly they are broken anyway because the server sends HSTS header for all subdomains.

**Comment by J0WI** [comment]
For Riseup we also have:
list.riseup.net
user.riseup.net
mail.riseup.net
crabgrass.riseup.net

404 on https:
dev.riseup.net

Mismatch:
vtolerance.riseup.net

Refused:
george.riseup.net

Sadly they are broken anyway because the server sends HSTS header for all subdomains.

Please open separate PRs next time.

File: src/chrome/content/rules/Bitmask.net.xml [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
-
-		- www.demo.bitmask.net
-		- www.dl.bitmask.net
-		- www.bitmask.net
+    Return an HSTS error:
+    - cdev.bitmask.net
+    - unstable.bitmask.net
 
 -->
 <ruleset name="Bitmask.net (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/Bitmask.net.xml]: I think you can remove the partial keyword here.

**Comment by just1602** [comment]
@J0WI How do you find all the subdomains so easily? :P

**Comment by J0WI** [comment]
See https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md

File: src/chrome/content/rules/Bitmask.net.xml [hunk 2]
```diff
@@ -1,34 +1,22 @@
...
-
-
-	These altnames don't exist:
-
-		- www.demo.bitmask.net
-		- www.dl.bitmask.net
-		- www.bitmask.net
+    Return an HSTS error:
+    - cdev.bitmask.net
+    - unstable.bitmask.net
```
Comment (J0WI) [inline src/chrome/content/rules/Bitmask.net.xml]: Please also use tabs instead of spaces here.

**Comment by J0WI** [comment]
Could you also add a comment to Riseup about the broken HSTS? They may have a look...

File: src/chrome/content/rules/Riseup.xml [hunk 3]
```diff
@@ -1,24 +1,25 @@
 <!--
 	Insecure cookies are set for these domains:
-
 		- labs.riseup.net
 
+	return an HPKP error:
```
Comment (J0WI) [inline src/chrome/content/rules/Riseup.xml]: I get HSTS/mismatch, but there is no HPKP.
See also  george.riseup.net (refused) and  dev.riseup.net (404). Both would work over http.

File: src/chrome/content/rules/Riseup.xml [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
-	<target host="list.riseup.net" />
-	<target host="we.riseup.net" />
-	<target host="crabgrass.riseup.net" />
-	<target host="labs.riseup.net" />
-	<target host="pad.riseup.net" />
-	<target host="share.riseup.net" />
-	<target host="black.riseup.net" />
-	<target host="user.riseup.net" />
-	<target host="mail.riseup.net" />
+    <target host="riseup.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/Riseup.xml]: It seems that you accidentally turned tabs in to spaces.

**Comment by J0WI** [comment]
All right, thanks!

**Pull request merged**